Makefile

```diff
@@ -177,10 +187,11 @@ help:  # Display this help
 ##@ generate:
 
 ALL_GENERATE_MODULES = core kubeadm-bootstrap kubeadm-control-plane
+ALL_GENERATE_OPENAPI = api/v1beta1 exp/runtime/hooks/api/v1alpha1 exp/runtime/hooks/api/v1alpha2 exp/runtime/hooks/api/v1alpha3
```

Comment:
```suggestion
```
Should be unused now and can be removed

Reply:
Yes, was not able to finish it last week :-) latest commit removes it and also has now a working `make release-manifests`